tests/models/test_user.py

```diff
@@ -177,6 +177,13 @@ def test_can_be_merged_both_addons(self):
 
         assert_false(self.user.can_be_merged)
 
+    def test_can_be_merged_delete_unmergable_addon(self):
+        self.user.add_addon('mergeable')
+        self.user.add_addon('unmergeable')
+        self.user.delete_addon('unmergable')
+
+        assert_false(self.user.can_be_merged)
```

Comment:
Shouldn't this test assert that the user can be merged after the unmergeable addon is removed?